#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////
# ------------------ Fastfusion Parameters ----------------------------
#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////


# Desired Threading
threadFusion : true
threadMeshing : true

# Scales
imageScale : 5000.0
scale : 0.006
distanceThreshold : 0.016

# Depth Consistency Checks
depthConsistencyChecks : 0

# Mesh Saving (location + desired Name without ending)
saveMesh : true
fileLocation : /home/karrer/test #/home/karrer/IROS2016/Evaluation/Vicon/Realsense/Desk/desk_6
# TF Id's
world_id : /world
cam_id : /depth
use_slam : false


# Define if using Time of flight sensor or not
use_pmd : true
depth_noise: true
